internal/servers/controller/handlers/outgoing_interceptor.go

```diff
@@ -18,9 +20,25 @@ func OutgoingInterceptor(ctx context.Context, w http.ResponseWriter, m proto.Mes
 	m = m.ProtoReflect().Interface()
```

Comment:
nil checks?

Reply:
Not sure what the contract is here, so not modifying what's there.